K2Bridge/Visitors/Aggregations/RangeAggregationVisitor.cs

```diff
@@ -59,10 +59,11 @@ public void Visit(RangeAggregation rangeAggregation)
 
             // Bucket expression:
             // >> count() by ['2'] | order by ['2'] asc
-            var bucketExpression = new StringBuilder();
+            var bucketExpression = $"{rangeAggregation.Metric} by {EncodeKustoField(rangeAggregation.Key)}";
 
-            bucketExpression.Append($"{rangeAggregation.Metric} by {EncodeKustoField(rangeAggregation.Key)}");
-            bucketExpression.Append($"{KustoQLOperators.CommandSeparator} {KustoQLOperators.OrderBy} {EncodeKustoField(rangeAggregation.Key)} asc");
+            // OrderBy expression:
+            // >> | order by ['2'] asc
+            var orderByExpression = $"{KustoQLOperators.CommandSeparator} {KustoQLOperators.OrderBy} {EncodeKustoField(rangeAggregation.Key)} asc";
```

Comment:
```suggestion
            var orderByExpression = $"{KustoQLOperators.CommandSeparator} {KustoQLOperators.OrderBy} {EncodeKustoField(rangeAggregation.Key)} asc";
```

 why 'asc' is hard coded?

Reply:
Yes. Seems the case for range, dateRange and Filters. Suggest we manage this point in a separate PR. FYI @tomconte.